src/TestEnvironment.Docker.Containers.Elasticsearch/ElasticsearchContainerWaiter.cs

```diff
@@ -7,30 +7,25 @@
 
 namespace TestEnvironment.Docker.Containers.Elasticsearch
 {
-    public class ElasticsearchContainerWaiter : IContainerWaiter<ElasticsearchContainer>
+    public class ElasticsearchContainerWaiter : BaseContainerWaiter<ElasticsearchContainer>
     {
-        private readonly ILogger _logger;
-
         public ElasticsearchContainerWaiter(ILogger logger = null)
+            : base(logger)
         {
-            _logger = logger;
         }
 
-        public async Task<bool> Wait(ElasticsearchContainer container, CancellationToken cancellationToken = default)
+        protected override async Task<bool> PerformCheckAsync(ElasticsearchContainer container,
```

Comment:
Make it single-line or move this parameter to the new line

Reply:
Fixed